addons/pos_restaurant/static/src/overrides/models/pos_store.js

```diff
@@ -44,59 +44,13 @@ patch(PosStore.prototype, {
         }
         return super.onDeleteOrder(...arguments);
     },
-    async getTableOrderCount() {
-        const result = await this.data.call(
-            "pos.config",
-            "get_tables_order_count_and_printing_changes",
-            [this.config.id]
-        );
-        this.ws_syncTableCount({ order_count: result, table_ids: [] });
-        this.bus.subscribe("TABLE_ORDER_COUNT", this.ws_syncTableCount.bind(this));
-    },
     // using the same floorplan.
     async ws_syncTableCount(data) {
-        const missingTable = data["order_count"].find(
-            (table) => !(table.id in this.models["restaurant.table"].getAllBy("id"))
-        );
-
-        if (missingTable) {
-            const response = await this.data.searchRead("restaurant.floor", [
-                ["pos_config_ids", "in", this.config.id],
-            ]);
-
-            const table_ids = response.map((floor) => floor.raw.table_ids).flat();
-            await this.data.read("restaurant.table", table_ids);
-        }
-        const tableByIds = this.models["restaurant.table"].getAllBy("id");
-        const tables = data["table_ids"].map((id) => this.models["restaurant.table"].get(id));
-        const draftTableOrders = [
-            ...new Set(
-                tables
-                    .map((t) => t["<-pos.order.table_id"])
-                    .flat()
-                    .filter((o) => !o.finalized)
-            ),
-        ];
-        await this.loadServerOrders([
-            "|",
-
-            // draft table orders in the server
-            ["state", "=", "draft"],
-
-            // draft table orders in client but paid in the server
-            "&",
-            ["state", "in", ["paid", "invoiced"]],
-            ["id", "in", draftTableOrders.map((t) => t.id)],
-
-            ["config_id", "in", [...this.config.raw.trusted_config_ids, this.config.id]],
-            ["table_id", "in", data["table_ids"]],
-            ["session_id", "=", odoo.pos_session_id],
-        ]);
-        for (const table of data["order_count"]) {
-            tableByIds[table.id].uiState.changeCount = table.changes;
-            tableByIds[table.id].uiState.orderCount = table.orders;
-            tableByIds[table.id].uiState.skipCount = table.skip_changes;
-        }
```

Comment:
This is from a recent bug fix and you are removing it. Did you test this change?